src/components/PersonLink.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  name?: string,
+  hasParent?: Person,
+};
+
+export const PersonLink: React.FC<Props> = ({
+  name,
+  hasParent,
```

Comment:
Maybe just parent? And destructure this inside the component, please
```suggestion
  parent,
```